index.html

```diff
@@ -0,0 +1,48 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Document</title>
+    <link rel="stylesheet" href="style.css">
+</head>
+<body>
+
+    <nav>
+
+        <label class="logo">Gerald</label>
+        <u>
+        <img src="images/hamburger-icon.png " id="menu" />
+        </u>
+    </nav>
```

Comment:
- [ ] Please add the `header` tag as a container and wrap it with the whole code because typically the `header` tag is used as a page header, it contains the site's title or logo (or both) and the main site navigation. [Check this link](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/header) for more info. Check below :point_down: my suggestions for more clarification :slightly_smiling_face: 

```suggestion
<header>
    <nav>

        <label class="logo">Gerald</label>
        <u>
        <img src="images/hamburger-icon.png " id="menu" />
        </u>
    </nav>
    </header>
```